cmake/Modules/CocosBuildHelpers.cmake

```diff
@@ -1,5 +1,29 @@
 include(CMakeParseArguments)
 
+# copy libs to static libs folder
+# error function, have bug
```

Comment:
What does it mean? Is this function used?

Reply:
it's a tool function, the purpose is support the unfinished feature: prebuilt, it's not been used for now